doc/importexport.rst

```diff
@@ -76,8 +76,114 @@ There are no parameters for this method other than the file path to import.
 
     result = cq.importers.importStep('/path/to/step/block.stp')
 
+Exporting STEP
+###############
+
+This section covers exporting CadQuery Workplane objects to STEP. For exporting assemblies to STEP, see the next section.
+
+Default
+--------
+
+The exporters module handles exporting Workplane objects to STEP. It is not necessary to set the export type explicitly
+since it will be determined from the file extension. Below is an example.
+
+.. code-block:: python
```

Comment:
An empty line must follow the start of each code-block to avoid errors like the following when building the docs: `ERROR: Error in "code-block" directive:
maximum 1 argument(s) allowed, 11 supplied.`